### DISCUSSION TITLE
problem with SMB share

### DISCUSSION THREAD

**Discussion by wambly542** [automatic-ripping-machine/automatic-ripping-machine#540] [General]
I can't seem to get my SMB share to work as my media folder. I have arm running in docker in a vm on a windows 10 box. I have mounted a SMB share from my NAS and checked that all permissions(read/write) were working from both ends. 

When I charge the paths in arm.yaml to my SMB mounted folder I get "There was a problem accessing the ARM folders. Please make sure you have setup ARM
. Setup can be started by visiting [setup page] ARM will not work correctly until you have added an admin account." in red on the home page in the web ui. When I change the paths back to the default locations the message goes away.

Am I missing something? 

Thanks!
![Home - ARM](https://user-images.githubusercontent.com/102460194/180600197-859ba137-4313-4e4a-bd46-26a2b163809f.png)
![Settings - ARM](https://user-images.githubusercontent.com/102460194/180600199-4726544a-3cb2-4e1f-9994-9d511b795d04.png)

**Comment by 1337-server** [comment]
https://github.com/automatic-ripping-machine/automatic-ripping-machine/wiki/Docker-Troubleshooting#changing-where-the-docker-puts-files

**Reply by wambly542** [reply]
Yep that would do it. I guess I need to read more lol. Thanks!

**Reply by maw74656** [reply]
Older thread, but I've read that section and not found any clarity.  What exactly did you do to resolve?

**Comment by aclater** [comment]
I would not put the raw and transcode on a networked file system - those should be direct attached or it is going to take forever.
